### PR TITLE
[iccpd] Fix uninitialized variable.

### PR DESCRIPTION
**Pull request by chenkelly**
**- Why I did it**
To declare *tb[] but do not initialize it, it might be very risky.
We get iccpd exception during processing arp/nd event.
**- How I did it**
To declare and initialize *tb[] variable.
**- How to verify it**
Test it based MC-LAG L2 scenario in HLD doc.

### CODE REVIEW

**Comment by jleveque** [comment]
Retest vs please

**Comment by lguohan** [comment]
retest mellanox please

**Comment by lguohan** [comment]
retest vsimage please

File: src/iccpd/src/iccp_ifm.c [hunk 1]
```diff
@@ -502,7 +502,7 @@ int do_one_neigh_request(struct nlmsghdr *n)
 {
     struct ndmsg *ndm = NLMSG_DATA(n);
     int len = n->nlmsg_len;
-    struct rtattr * tb[NDA_MAX + 1];
+    struct rtattr *tb[NDA_MAX + 1] = {NULL};
```
Comment (jleveque) [inline src/iccpd/src/iccp_ifm.c:505]: It looks like line 969 is also not zeroing out the struct. Suggest making the same change there, as well.

Also, suggest changing from `= {NULL};` to `= { 0 };`. Theoretically the behavior is the same, but the latter is more commonly used.

**Pull request merged**